addons/addons.json

```diff
@@ -1,4 +1,5 @@
 [
+  "necropost-highlighter",
```

Comment:
Should generally be at the end where new userscripts are supposed to be

Reply:
This would be a good clarification for Step 7 of [Getting Started](https://scratchaddons.com/docs/develop/getting-started/creating-an-addon/).